biom/table.py

```diff
@@ -2697,6 +2697,10 @@ def subsample(self, n, axis='sample', by_id=False):
             matrix (e.g., rarefaction). If `True`, the subsampling is based on
             the IDs (e.g., fetch a random subset of samples). Default is
             `False`.
+        with_replacement : boolean, optional
+            If `False` (default), subsample without replacement. If `True`,
+            resample with replacement via the multinomial distribution.
+            If `True` and `by_id=True`, a ValueError is raised.
```

Comment:
Had a comment here but it seemed to drop -- apologize if this is a duplicate. Can you please add the exception note to the `Raises` section instead?

Reply:
I think this should be the case now.